ambari-server/src/main/java/org/apache/ambari/server/orm/DBAccessorImpl.java

```diff
@@ -76,6 +77,11 @@
 @Singleton
 public class DBAccessorImpl implements DBAccessor {
   private static final Logger LOG = LoggerFactory.getLogger(DBAccessorImpl.class);
+  public static final String USER = "user";
+  public static final String PASSWORD = "password";
+  public static final String NULL_CATALOG_MEANS_CURRENT = "nullCatalogMeansCurrent";
+  public static final String TRUE = "true";
+  public static final int SUPPORT_CONNECTOR_VERSION = 5;
```

Comment:
For the forward-port purpose, this is good but otherwise we can make them all private in separate commit.

Reply:
Yep!